spec/features/merchants/dashboard_spec.rb

```diff
@@ -103,5 +103,14 @@
       expect(page).to have_content(@invoice_4.created_at.strftime("%A, %B %d,%Y"))
       expect(page).to have_content(@invoice_5.created_at.strftime("%A, %B %d,%Y"))
     end
+    it '#US36 lists items oldest to newest' do
```

Comment:
Great job testing for order here -  would love to go over the orderly gem more with you if you have time too